### DISCUSSION TITLE
Was kommt als Nächstes?

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#3523] [Ideas]
Angekündigt kürzlich im [Blog](https://solectrus.de/blog/2024-09-23-version-0-17/), sicherheitshalber auch hier:

Bei der Planung der weiteren Entwicklungsarbeit möchte ich gerne die Nutzer einbeziehen.

Es gibt derzeitig 45 offene Issues. Einige davon sind größere Features, andere sind kleinere Verbesserungen. Hier ein paar Beispiele größerer Themen, die möglicherweise für viele interessant sind:

- #3396
- #3517
- #2407
- #2416

Issues lassen sich bewerten (aka "Like") und dann als sortierte Liste ausgeben. Das ist eine einfache Möglichkeit, um zu sehen, welche Features die Nutzer am meisten interessieren.

Ich möchte alle Nutzer aufrufen, sich die Liste anzusehen und die Issues zu bewerten, die am wichtigsten erscheinen. Das hilft mir, die Prioritäten richtig zu setzen.

Zu beachten:

- Die Bewertung erfolgt im einzelnen Issue. Aus der Liste heraus ist also das zu bewertende Issue aufzurufen.
- Es wird nur die Bewertung des **ersten** Beitrags berücksichtigt, also nicht die der Kommentare darunter.
- Relevant ist die Bewertung mit "Daumen hoch" (👍). Andere Reaktionen werden in der Auswertung nicht berücksichtigt (auch nicht "Daumen runter"). Eine Bewertung kann aber zurückgezogen werden, was auch berücksichtigt wird.
- Ergänzende Kommentare (im Issue) sind natürlich willkommen.
- Neue Issues können gerne erstellt werden. Bitte vorher die Suchfunktion verwenden, um Duplikate zu vermeiden.
- Um eine Bewertung abzugeben, musst du bei GitHub angemeldet sein. Das ist kostenlos und dauert nur wenige Minuten.
- Man muss kein Sponsor sein, um abzustimmen. Jeder Nutzer ist eingeladen, sich zu beteiligen.

Genug der Vorrede, hier der Link zu den Issues, über die abgestimmt werden kann:
https://github.com/solectrus/solectrus/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by ledermann** [comment]
Wie es mittlerweile aussieht, ist #3396 (Definieren zusätzlicher Verbraucher) ganz oben auf der Wunschliste - auch wenn nur wenige der SOLECTRONAUTEN an der Abstimmung teilgenommen haben (möglicherweise schreckt die GitHub-Registrierung viele ab). Wenn sich also nichts grundlegend ändert, nehme ich mir das als Nächstes vor.

Eine Sache möchte ich aber noch zuvor fertig bekommen, und das ist #3533. Das ist eine größere interne Änderung, die erhebliche positive Auswirkungen auf Genauigkeit und Performance aller Auswertungen hat. Daran arbeite ich derzeitig.

Die Möglichkeit, Issues zu bewerten, bleibt ausdrücklich bestehen. Es kommen ja auch immer wieder neue hinzu.

**Comment by kdk71** [comment]
Hallo Georg,

wäre es möglich auch eine Modbus-Anbindung zu integrieren? Viele Wärmepumpen nutzen dieses Protokoll.

Viele Grüße,
Klaus

**Reply by ledermann** [reply]
Möglich ja, aber unwahrscheinlich. Ich selbst habe kein einziges Gerät im Haus, das auf Modbus angewiesen wäre.

Meine Empfehlung daher: Umweg über HomeAssistant oder ioBroker nehmen. Dafür gibt es unzählige Adapter, mit denen die Messwerte eingesammelt werden können. Dann die Messwerte per MQTT von dort über den [MQTT-Collector](https://docs.solectrus.de/referenz/mqtt-collector/) abholen.
